clients/web/src/templates/widgets/uploadWidgetMixins.pug

```diff
@@ -17,7 +17,7 @@ mixin g-upload-widget-browse-or-drop
     else
       |  Browse or drop files here
   .form-group.hide
-    if (parentType === 'file')
+    if (parentType === 'file' || !multiFile)
```

Comment:
The upload widget says "files" in two places, but in this case, it should say "file" as only one is allowed.

<img width="596" alt="screen shot 2016-11-13 at 1 49 06 pm" src="https://cloud.githubusercontent.com/assets/595023/20248041/fa3cb866-a9a9-11e6-8c33-de378cd1c8ef.png">